tests/js/setup.js

```diff
@@ -370,10 +370,20 @@ window.TestStubs = {
     }
   },
 
+  Event: params => {
+    return {
+      ...params,
+      id: '1',
+      message: 'ApiException',
+      groupID: '1',
+      eventID: '12345',
```

Comment:
if you want your inputs to override, put ...params last i think

Reply:
oops good catch